.github/workflows/chroma-integration-test.yml

```diff
@@ -14,7 +14,7 @@ jobs:
   test:
     strategy:
       matrix:
-        python: ['3.10']
+        python: ['3.7', '3.8', '3.9', '3.10']
```

Comment:
This is good just to be sure, but I think we *might* be able to get away with not testing cross-version for integration tests.

Rationale: the actual Chroma server runs in Docker, which is always going to use a fixed Chroma version (i.e, not dependent on the test environment.) The integration tests do exercise the FastAPI client API, but that's *also* exercised as part of the standard tests.

So I think we can probably skip the cross-version stuff for integration tests.

Reply:
Agreed, will change before merge.